### PR TITLE
Create book by isbn

### PR DESCRIPTION
**Pull request by rafaelrpbelo**
- Catcher to get a book by an isbn.
- Create a book by an isbn through the GoogleBooks API.

### CODE REVIEW

File: spec/catchers/googlebooks_spec.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 require 'rails_helper'
 
-describe 'GoogleBooks' do
+describe GoogleBooks do
   context 'catch a book by' do
     it 'isbn' do
       VCR.use_cassette('search-isbn') do
-        book = GoogleBooks.search('isbn:9781118084786').first
+        book = described_class.search('isbn:9781118084786').first
```
Comment (dmitryrck) [inline spec/catchers/googlebooks_spec.rb]: You don't need to change anything in this file :/

File: spec/business/book_creator_spec.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  it 'create a book by isbn' do
+    VCR.use_cassette('search-isbn') do
+      book = described_class.create_by_isbn('9781118084786')
+
+      expect(book).to be_a Book
+      expect(book.persisted?).to be true
+      expect(book.isbn).to eq '9781118084786'
+      expect(book.title).to eq 'Ruby on Rails For Dummies'
+      expect(book.authors).to eq 'Barry Burd'
+      expect(book.year_published).to eq 2011
```
Comment (dmitryrck) [inline spec/business/book_creator_spec.rb:32]: It is a better idea split each `expect` ;)

File: spec/catchers/googlebooks_spec.rb (hunk 1 above)
Comment (rafaelrpbelo) [inline spec/catchers/googlebooks_spec.rb]: Why? described_class is so much cute '-'

Comment (dmitryrck) [inline spec/catchers/googlebooks_spec.rb]: ok, but why change a code only to be cute?

In other words: this code is not related with your PR :| 
- Your PR: "Create book by isbn";
- This file changes: "be cute".

Is it related?

File: spec/business/book_creator_spec.rb (hunk 2 above)
Comment (rafaelrpbelo) [inline spec/business/book_creator_spec.rb:32]: So how can I split this?
Do you want I put each expect in a separate sample?

File: spec/catchers/googlebooks_spec.rb (hunk 1 above)
Comment (rafaelrpbelo) [inline spec/catchers/googlebooks_spec.rb]: Ok Sir, I'll remove this. '-'

File: spec/business/book_creator_spec.rb (hunk 2 above)
Comment (dmitryrck) [inline spec/business/book_creator_spec.rb:32]: @rafaelrpbelo : https://gist.github.com/dmitryrck/14828a332b6274514bdf what do you think about?

Comment (rafaelrpbelo) [inline spec/business/book_creator_spec.rb:32]: Yes, so much better! Thx ;)

File: spec/catchers/catchers_spec.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+describe 'Catchers' do
+  context 'GoogleBook' do
+    subject { Catchers::GoogleBook }
+
+    it 'should to catch a book' do
+      VCR.use_cassette('search-isbn') do
+        catched_book = subject.new('9781118084786').catch
+
+        expect(catched_book).to be_a GoogleBooks::Item
+        expect(catched_book.title).to eq 'Ruby on Rails For Dummies'
```
Comment (dmitryrck) [inline spec/catchers/catchers_spec.rb]: You can split this too :)

File: spec/catchers/googlebooks_spec.rb (hunk 1 above)
Comment (dmitryrck) [inline spec/catchers/googlebooks_spec.rb]: I thank you :+1:

**Comment by dmitryrck** [comment]
Thank you @rafaelrpbelo :+1:
